### DISCUSSION TITLE
Need suggestion for the SEO using free hosting service of Vercel

### DISCUSSION THREAD

**Discussion by mostafijmf** [vercel/community#3951] [Feature Question] [answered]
### Question

Hi, this is Mostafij,
I need your help. I've created a blog website using Next.js and I wanna rank this on Google.
Will using Vercel free plan hosting be good for SEO or have a negative impact?
Thanks

**Comment by swarnava** [comment, marked as answer]
Hi,

There shouldn't be any impact for SEO if you deploy your website in Vercel. You can review SEO score using third party tool for more information on this.

**Reply by mostafijmf** [reply]
Thank you!
